README.md

```diff
@@ -26,6 +26,28 @@ This is the dependency you need:
 Read the [Javadoc](http://www.javadoc.io/doc/com.artipie/asto)
 for more technical details.
 
+Create a `hello.txt` file with `"Hello World!"` content on file-system-based
```

Comment:
@Sammers21 maybe you can here an intro text, explaining what this `Storage` is for? A few sentences. Now this sounds too cold, without an intro.

Reply:
That is a good idea. Just done